### DISCUSSION TITLE
和 Rick 聊 2023 开源之夏【双周，非官方】

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [LinuxSuRen/open-source-best-practice#141] [General]
2022 年，我在 KubeSphere 社区以 mentor 的角色参与开源之夏时，会组织双周例会，以解答学生的疑问。而且，当时开源之夏官方也会组织双周例会。这是一种非常好的交流互动的方式，如果有 10+ 小伙伴感兴趣的话，我会组织起来。

* 会议形式：腾讯会议
* 是否录屏：会录屏，并上传哔哩哔哩
* 是否会涉及特定项目：不会，但我会解答我了解的技术，其他参会者也可以帮忙解决相关问题
* 主持人擅长领域：云原生，Golang，Java，Kubernetes

点击链接加入腾讯会议：
https://meeting.tencent.com/dm/bnEsskuvnmyF

时间，周日晚九点，23 年 5 月 21 日开始（双周）

每次会议录屏链接会放在本页面。

* [23/5/21](https://www.bilibili.com/video/BV1Y24y1N72E/?vd_source=ca1041974e9b44975cc4d5355ad7001f)

**Comment by erdengk** [comment]
报名+1

**Comment by OrezzerO** [comment]
+1

**Comment by wjsvec** [comment]
这周没有了吗🥲

**Reply by LinuxSuRen** [reply]
抱歉，刚到家，有点事情耽搁了。

**Comment by LinuxSuRen** [comment]
非常抱歉了，个人精力实在跟不上 host 会议，以及对该主题感兴趣的人很少，关闭这个会议以及主题。
